src/bindings.cpp

```diff
@@ -249,6 +363,9 @@ void bind_enums_and_constants(py::module &m)
 
     //NATIVEINT function
     m.def("get_native_int", &get_native_int);
+
+    // Params
+    py::class_<Params>(m, "Params");
```

Comment:
Is this meant to return the parameters object? What happens if the user tries to interact with it? Do we have code supporting that ?

Reply:
That's the class behind CCParams. The user doesn't interact with Params directly, but I had to expose it because of the inheritance relationship between them.

So, the user interface will be focused on the three classes: CCParamsBGVRNS, CCParamsBGVRNS and CCParamsCKKSRNS

Reply:
Noted! I'm sure you've already thought of this, but if the user doesn't interact with this directly can we hide it behind an interface?

Reply:
I can try remove this part, but I remember having an error, and pybind requiring this line, exposing the base class for CCParams.

Maybe it's because the way I have binded CCParams, specifying the inheritance. If I change this part I think I can remove Params.

Reply:
Done! My last commit addressed this. I'll close the PR